### PR TITLE
Add language support to Forgot Password

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/appstore/modules/forgot-password.md [hunk 1]
```diff
@@ -95,11 +95,15 @@ In these instructions, it is assumed that your main module is `MyFirstModule`. I
...
-    * Click **SMTP settings** to configure or validate SMTP settings for the [Email Connector](/appstore/connectors/email-connector/) or [Email with Templates](/appstore/modules/email-with-templates/) module (depending on the version of the Forgot Password module)
-    * Click **Reset email template** and provide the details for the email sent when an end-user has forgotten their password
+    * Click **SMTP settings** to configure or validate SMTP settings for the [Email Connector](/appstore/connectors/email-connector/) or [Email with Templates](/appstore/modules/email-with-templates/) module (depending on the version of the Forgot Password module).
+    * Click **Reset email template** and provide the details for the email sent when an end-user has forgotten their password.
+
+        For version 5.3.0 and above, you can link each template to a [language](/refguide/language-settings/) you have added to your app. If there is no template explicitly associated with the end-user's language, the template which is not associated with a language will be used.
+
         {{< figure src="/attachments/appstore/modules/forgot-password/email-template.png" >}}
```
Comment (MarkvanMents) [inline content/en/docs/appstore/modules/forgot-password.md]: This image does not show where the language setting is. Do we need a new (high quality) image?

Comment (SharishmaPodduturi999) [reply content/en/docs/appstore/modules/forgot-password.md]: Language can be set by opening the template.
image

File: content/en/docs/appstore/modules/forgot-password.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ tags: ["marketplace", "marketplace component", "forgot password", "password", "l
 
 The [Forgot Password](https://marketplace.mendix.com/link/component/1296/) module enables the users to sign up and also to reset their password. It works with the local accounts which are managed within your Mendix app. These are the best choice when you do not want to use a Single Sign On (SSO) solution to integrate with your existing Identity and Access Management (IAM) infrastructure.
 
-This module allows the end-user to enter their email address, and an email will be sent with a confirmation link. The end-user then opens the link and gets the option to reset their password in both scenarios (Sign up and Forgot password). In the sign up case, the end-user will also be asked to provide their name.
+This module allows the end-user to enter their email address, and an email will be sent with a confirmation link. The end-user then opens the link and gets the option to reset their password in both scenarios (Sign up and Forgot password). In the sign up case, the end-user will also be asked to provide their name. For version 5.3.0 of the module and above, you can specify different templates to be sent, depending on the language in which the end-user has chosen to see the app.
```
Comment (SharishmaPodduturi999) [inline content/en/docs/appstore/modules/forgot-password.md]: `For version 5.3.0 of the module and above, you can specify different templates to be sent, depending on the language in which the end-user has chosen to see the app.`

Can we change the above line?
For version 5.3.0 of the module and above, you can specify different templates to be sent, depending on the language set on end-user account.
Languages can be set from the app(studio pro)

File: content/en/docs/appstore/modules/forgot-password.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/forgot-password.md]: I can't see how customers can get to a page that looks like this. In our [current documentation](https://docs.mendix.com/appstore/modules/forgot-password/#installing), the **Placeholder** is a tab on the setup screen.
image

 I also can't see anything about it in the Email Connector documentation: https://docs.mendix.com/appstore/connectors/email-connector/#43-using-email-templates

Comment (SharishmaPodduturi999) [reply content/en/docs/appstore/modules/forgot-password.md]: image
The UI of Reset Password Email has been changed.
On clicking on Create or Edit button, users can open specific template where they can see the language at the bottom

Comment (SharishmaPodduturi999) [reply content/en/docs/appstore/modules/forgot-password.md]: We are using Snippet from EmailConnector in our module

File: content/en/docs/appstore/modules/forgot-password.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/forgot-password.md]: This level of technical detail is described on line 105. I don't think it is necessary in the introduction.

@SharishmaPodduturi999 - can I confirm that this is just version 5 of the module, and that we aren't retrofitting it into versions 4 and 3

File: content/en/docs/appstore/modules/forgot-password.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/modules/forgot-password.md]: I've updated this in commit https://github.com/mendix/docs/pull/6037/commits/fa34d81a49aeea992c13a0644c1f18543c91ae82

**Comment by MarkvanMents** [comment]
Released

**Pull request merged**